### PR TITLE
kubelet_port not needed in prometheus kubernetes config anymore

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

**Comment by jimmidyson** [comment]
LGTM

File: prometheus/src/main/resources/prometheus.yml [hunk 1]
```diff
@@ -30,7 +30,6 @@ scrape_configs:
   - api_servers:
     - 'https://kubernetes.default.svc'
     in_cluster: true
-    kubelet_port: 10250
```
Comment (rawlingsj) [inline prometheus/src/main/resources/prometheus.yml]: @jimmidyson you happy for this to go in? I've tested on GKE and it works but perhaps a test on openshift is needed too?

Comment (jimmidyson) [inline prometheus/src/main/resources/prometheus.yml]: I'm happy. The port discovery should have been there for quite a long time.

Comment (rawlingsj) [inline prometheus/src/main/resources/prometheus.yml]: perfect - thanks

**Pull request merged**